src/components/Objects/Form/__tests__/FormCaption.test.js

```diff
@@ -0,0 +1,104 @@
+import React from 'react';
+import { render } from 'test-utils';
+import FormConstants from '../../../../constants/form-constants';
+
+import FormCaption from '../FormCaption';
+
+const CAPTION = 'Caption';
+const SHOW_CAPTION = true;
+const HIDE_CAPTION = false;
+const HAS_ERROR = true;
```

Comment:
couldn't you use !Has_error instead of a second variable?

Reply:
very true, probably can clean all that up. On it 😎